src/components/chat-input/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import colors from "tailwindcss/colors";
+import { SendArrow } from "@pb/components/icons";
+
+type ChatInputProps = {
+  onSendMessage: (message: string) => void;
+};
+
+const ChatInput = ({ onSendMessage }: ChatInputProps) => {
+  const [message, setMessage] = useState("");
+
+  const handleMessageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setMessage(e.target.value);
+  };
+
+  const handleSendMessage = () => {
+    if (message.trim() !== "") {
```

Comment:
message defaults to empty string, it would never be undefined

Reply:
@OsamaAlmaani - I didn't mean the value of it when it is already sourced from the state. Instead, I meant when we integrate it with the back-end (BE), such as when users click on the edit mode. In this scenario, we set the value of the message to be equal to it, but there is a possibility that it could be returned as null or undefined.